setup.py

```diff
@@ -27,7 +27,7 @@ def read(fname):
     install_requires=(
         'Django>=1.5',
         'easy-thumbnails>=1.0',
-        'django-mptt>=0.6',
+        'django-mptt>=0.6,<0.8',
```

Comment:
@yakky 0.8 release drops ``Django <1.8`` AND ``Python 2.6`` and I don't thinks that this is correct to limit  mptt version for python 2.7 for example. Maybe it is better to adjust ``README``'s and limit mptt version only for ``Python<2.7``?
something like https://github.com/aldryn/aldryn-events/blob/master/setup.py#L35